frame/support/src/dispatch.rs

```diff
@@ -433,6 +433,10 @@ impl PerDispatchClass<Weight> {
 
 	/// Add some weight of a specific dispatch class, saturating at the numeric bounds of `Weight`.
 	pub fn add(&mut self, weight: Weight, class: DispatchClass) {
```

Comment:
`add` doesn't have this meaning in the arithmetic traits (it would be more akin to `AddAssign`). is it worth changing this now before it gets used too much?

Reply:
Yea it look wrong, so we should change it anyway.

Reply:
Fixed in https://github.com/paritytech/substrate/pull/13164/commits/2dc79da5b354e5e564156142e2263628a86deb29